uhdm-plugin/UhdmAst.cc

```diff
@@ -609,9 +609,10 @@ void UhdmAst::process_port() {
 		} else if (n == vpiOutput) {
 			current_node->is_output = true;
 		} else if (n == vpiInout) {
-			current_node->is_input = true;
-			current_node->is_output = true;
+			current_node->is_input = current_node->is_output = true;
 		}
+	} else {
+		current_node->is_input = current_node->is_output = true;
```

Comment:
This looks weird. Why we want to set input/output when Surelog reports no direction?

This is because IEEE standard says so? If yes, I think it should be fixed in Surelog.